x-pack/test/plugin_api_integration/plugins/event_log/server/init_routes.ts

```diff
@@ -0,0 +1,286 @@
+      req: KibanaRequest<any, any, any, any>,
+      res: KibanaResponseFactory
+    ): Promise<IKibanaResponse<any>> {
+      const { provider } = req.params as { provider: string };
+
+      logger.info(`test if get all provider actions is registered`);
+      return res.ok({
+        body: { actions: [...(eventLogService.getProviderActions().get(provider) ?? [])] },
+      });
+    }
+  );
+};
+
+export const getLoggerRoute = (
+  router: IRouter,
+  eventLogService: IEventLogService,
+  logger: Logger
+) => {
+  router.get(
+    {
+      path: `/api/log_event_fixture/getEventLogger/{event}`,
+      validate: {
+        params: (value: any, { ok }: RouteValidationResultFactory) => ok(value),
+      },
+    },
+    async function(
+      context: RequestHandlerContext,
+      req: KibanaRequest<any, any, any, any>,
+      res: KibanaResponseFactory
+    ): Promise<IKibanaResponse<any>> {
+      const { event } = req.params as { event: string };
+      logger.info(`test get event logger for event: ${event}`);
+
+      return res.ok({
+        body: { eventLogger: eventLogService.getLogger({ event: { provider: event } }) },
+      });
+    }
+  );
+};
+
+export const isIndexingEntriesRoute = (
+  router: IRouter,
+  eventLogService: IEventLogService,
+  logger: Logger
+) => {
+  router.get(
+    {
+      path: `/api/log_event_fixture/isIndexingEntries`,
+      validate: {},
+    },
+    async function(
+      context: RequestHandlerContext,
+      req: KibanaRequest<any, any, any, any>,
+      res: KibanaResponseFactory
+    ): Promise<IKibanaResponse<any>> {
+      logger.info(`test if event logger is indexing entries`);
+      return res.ok({ body: { isIndexingEntries: eventLogService.isIndexingEntries() } });
+    }
+  );
+};
+
+export const isEventLogServiceEnabledRoute = (
+  router: IRouter,
+  eventLogService: IEventLogService,
+  logger: Logger
+) => {
+  router.get(
+    {
+      path: `/api/log_event_fixture/isEventLogServiceEnabled`,
+      validate: {},
+    },
+    async function(
+      context: RequestHandlerContext,
+      req: KibanaRequest<any, any, any, any>,
+      res: KibanaResponseFactory
+    ): Promise<IKibanaResponse<any>> {
+      logger.info(`test if event logger is enabled`);
+      return res.ok({ body: { isEnabled: eventLogService.isEnabled() } });
+    }
+  );
+};
+
+export const isEventLogServiceLoggingEntriesRoute = (
+  router: IRouter,
+  eventLogService: IEventLogService,
+  logger: Logger
+) => {
+  router.get(
+    {
+      path: `/api/log_event_fixture/isEventLogServiceLoggingEntries`,
+      validate: {},
+    },
+    async function(
+      context: RequestHandlerContext,
+      req: KibanaRequest<any, any, any, any>,
+      res: KibanaResponseFactory
+    ): Promise<IKibanaResponse<any>> {
+      logger.info(`test if event logger is logging entries`);
+      return res.ok({ body: { isLoggingEntries: eventLogService.isLoggingEntries() } });
+    }
+  );
+};
+
+export const startTimingEventLoggerRoute = (
```

Comment:
I don't think we need these routes - the start/stop timing methods on the logger are designed to be used to set the relevant timing information in a single event that gets logged.  I think we already have some jest tests for these, that's probably all we really need to test for them, I think.  Or we could have one of the test loggers fixtures write do some more realistic timings - I made a note of that in the `logEventRoute()` ^^^